src/include/processor/operator/persistent/export_db.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "catalog/catalog.h"
+#include "processor/operator/sink.h"
+#include "processor/result/result_set.h"
+
+namespace kuzu {
+namespace processor {
+
+class ExportDB : public PhysicalOperator {
+public:
+    ExportDB(std::string filePath, std::vector<std::string> tableNames,
+        std::vector<std::string> tableCyphers, std::vector<std::string> macroCyphers, uint32_t id,
+        const std::string& paramsString, std::vector<std::unique_ptr<PhysicalOperator>> children)
+        : PhysicalOperator{PhysicalOperatorType::EXPORT_DATABASE, id, paramsString},
+          filePath(filePath), tableNames(std::move(tableNames)),
+          tableCyphers(std::move(tableCyphers)), macroCyphers(macroCyphers) {
+        PhysicalOperator::children = std::move(children);
```

Comment:
The physical operator constructor takes in children as a parameter, you can pass the children to the constructor.